dev/run-tests.py

```diff
@@ -128,7 +128,7 @@ def determine_tags_to_exclude(changed_modules):
     tags = []
     for m in modules.all_modules:
         if m not in changed_modules:
-            tags += m.test_tags
+            tags += m.tags
```

Comment:
`m.tags` (née `m.test_tags`)  is a list of tags that identify tests that are to be excluded when that module hasn't changed. It's explained in dev/sparktestsupport/modules.py (where the field is defined).